utils/logging/log.go

```diff
@@ -67,7 +68,9 @@ func (l *log) Write(p []byte) (int, error) {
 // TODO: return errors here
 func (l *log) Stop() {
 	for _, wc := range l.wrappedCores {
-		_ = wc.Writer.Close()
+		if wc.Writer != os.Stdout && wc.Writer != os.Stderr {
```

Comment:
Can you explain this change?

Reply:
This was part of the merge conflict resolution. The only real change in this PR is https://github.com/ava-labs/avalanchego/pull/2375/commits/32986938957f7e2026b9e6e590d4cc0ccdf9d508

This change in particular originated from https://github.com/ava-labs/avalanchego/pull/2372